HAVE_SSIZE_T missing for libmemcachedprotocol

Trying to build pecl memcached extension on 32-bit fails

See https://koji.fedoraproject.org/koji/taskinfo?taskID=72457513

In config.log

```
In file included from conftest.c:28:
/usr/include/libmemcachedprotocol-0.0/handler.h:20:18: error: conflicting types for 'ssize_t'; have 'long int'
   20 | typedef long int ssize_t;
      |                  ^~~~~~~
In file included from /usr/include/libmemcachedprotocol-0.0/handler.h:18,
                 from conftest.c:28:
/usr/include/sys/types.h:108:19: note: previous declaration of 'ssize_t' with type 'ssize_t' {aka 'int'}
  108 | typedef __ssize_t ssize_t;
      |                   ^~~~~~~
conftest.c: In function 'main':
conftest.c:32:39: warning: variable 's_test_impl' set but not used [-Wunused-but-set-variable]
   32 | memcached_binary_protocol_callback_st s_test_impl;

```      |                                       ^~~~~~~~~~~


Diag:  `HAVE_SSIZE_T` is only defined in `libmemcached-1.0/configure.h`

--- a/include/libmemcachedprotocol-0.0/CMakeLists.txt
+++ b/include/libmemcachedprotocol-0.0/CMakeLists.txt
@@ -2,6 +2,7 @@
 install_public_headers(
         libmemcachedprotocol-0.0
 
+        @configure.h
         binary.h
         callback.h
         handler.h

--- a/include/libmemcachedprotocol-0.0/configure.h.in
+++ b/include/libmemcachedprotocol-0.0/configure.h.in
@@ -1,0 +1,19 @@
+/*
+    +--------------------------------------------------------------------+
+    | libmemcached-awesome - C/C++ Client Library for memcached          |
+    +--------------------------------------------------------------------+
+    | Redistribution and use in source and binary forms, with or without |
+    | modification, are permitted under the terms of the BSD license.    |
+    | You should have received a copy of the license in a bundled file   |
+    | named LICENSE; in case you did not receive a copy you can review   |
+    | the terms online at: https://opensource.org/licenses/BSD-3-Clause  |
+    +--------------------------------------------------------------------+
+    | Copyright (c) 2006-2014 Brian Aker   https://datadifferential.com/ |
+    | Copyright (c) 2020-2021 Michael Wallner        https://awesome.co/ |
+    +--------------------------------------------------------------------+
+*/
+
+#pragma once
+
+#cmakedefine HAVE_SSIZE_T 1
+

--- a/include/libmemcachedprotocol-0.0/handler.h
+++ b/include/libmemcachedprotocol-0.0/handler.h
@@ -15,6 +15,7 @@
 
 #pragma once
 
+#include "libmemcachedprotocol-0.0/configure.h"
 #include <sys/types.h>
 #ifndef HAVE_SSIZE_T
 typedef long int ssize_t;

